<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Analysis</class>
 <widget class="QWidget" name="Analysis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QLineEdit" name="yearLineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:3px;
border-width:1px;
border-style:solid;</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="placeholderText">
      <string>YYYY</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>575</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="drawButton">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover
{
background-color:  rgb(100, 182, 100);

}

QPushButton 
{
     font: 75  14pt &quot;Calibri&quot;;
	background-color: #1FDA9A;
	
	color: rgb(255, 255, 255);

	border-radius: 5px;

}</string>
     </property>
     <property name="text">
      <string>Plot</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QCustomPlot" name="customPlot" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBox">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>25</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Monthly</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Yearly</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
